### ISSUE TITLE
Discoverability of features

### ISSUE THREAD

**Issue by somas95** [UberWriter/uberwriter#69]
Currently we have a problem; some of the best features of UberWriter aren't easily discoverable. Such features are, at least, but not only:

- Inline preview
- Formatting of selected text with keyboard shortcuts

There're some ways to help users to find out this features, but they could be intrusive. Some of them are, from more intrusive and difficult to make to less intrusive and easier to do:

- A first time interactive tutorial (Glade does this)
- A "tips and tricks" window that opens with the program and can be deactivated (Very typical)
- A text based tutorial. We have one but it's so long that most users won't read it. (Emacs does this)

Either if we do one of these things or not, we should provide good documentation

**Comment by tlnagy** [comment]
> Inline preview

I looked for how to activate this feature and I have no idea. I would definitely appreciate a short tutorial. I think the 2nd option would work well.

**Comment by somas95** [comment]
Yeah, I'll go with that. In the meantime, ctrl + click on anything opens a preview of that element

**Comment by Frivolous2** [comment]
Can I just say that "tips and tricks" is very annoying and I don't think that's a good idea. I think a welcome wizard, with "Hi, this is uberwriter. You can format text with *these* and ##these, but you can also check out the keyboard shortcuts by clicking 'Keyboard shortcuts' in the menu. Have fun!" would be more appropriate and quick.
Of course, that would mean a simple graphic 'Keyboard shortcuts' needs to be implemented under a menu item.
Spitball example:
![image](https://user-images.githubusercontent.com/57101103/75610530-f1ee9700-5b09-11ea-9b2e-23b4b006412b.png)
